erpnext_com/www/docs/user/manual/en/setting-up/email/email-dropbox.md

```diff
@@ -0,0 +1,27 @@
+<!-- add-breadcrumbs -->
+
+# Email Dropbox
+
+When you receive emails from leads, customers, suppliers and other business partners, you may like to attach that to relevant records in ERPNext. You can save the email and upload it as an attachment. However, this is a cumbersome process. ERPNext offers a better way to handle this.
+
+Each record in ERPNext provides a unique email address to which you can forward emails. ERPNext will attach the email to that record.
+
+## 1. Enable Email Dropbox Feature
+
+In [Email Account](/docs/user/manual/en/setting-up/email/email-account), check 'Enable Email Dropbox'.
+
+<img class="screenshot" alt="Enable Email Dropbox" src="{{docs_base_url}}/assets/img/setup/email/enable_email_dropbox.png">
+
+## 2. Copy Unique Email Address in Any Record
+
+Once the Email Dropbox feature is enables, the unique email address becomes visible in all the records. You can click on the same and the address is copied to your clipboard. You can directly paste it in 'To', 'Cc' or 'Bcc' and send the email.
+
+<img class="screenshot" alt="Unique Email Address" src="{{docs_base_url}}/assets/img/setup/email/unique_email_address_dropbox.png">
+
+## 3. Email in Timeline
+
+Once the email is received, ERPNext attaches it to the appropriate record and it will be visible in the timeline of the record.
```

Comment:
I think "in that record's timeline" sounds more clearer in lesser words than "in the timeline of the record"

Reply:
updated...